app/src/main/java/waiter/Router.java

```diff
@@ -0,0 +1,31 @@
+package waiter;
+
+import java.util.Map;
+
+public class Router {
+
+    Map<String, Map<String, String>> routes;
+
+    public Router() {
+        this.routes = Map.ofEntries(
+                Map.entry("GET /simple_get", setRoute("200 OK", "Content-Length:", "")),
+                Map.entry("GET /simple_get_with_body", setRoute("200 OK", "Content-Length:", "Hello world"))
+        );
+    }
+
+    private Map<String, String> setRoute(String status, String headers, String body) {
```

Comment:
This method is named `setRoute`, but it takes the arguments `status`, `headers`, and `body` – all of which seem to be more concerned with the details of a response rather than the route with which they are associated.

Reply:
That is a really good point, I agree, maybe something like `setResponseFields`?

Reply:
Im gonna go with this for now, if we can find a better one I can always change it  in the future 😄

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/14/commits/79f9d4fd94a507209a16c6c92dc3b0231800b339)